submission/distributor.go

```diff
@@ -179,7 +181,22 @@ func incRspsCounter(logURL string, endpoint string, rspErr error) {
 			status = err.StatusCode
 		}
 	}
-	rspsCounter.Inc(logURL, endpoint, strconv.Itoa(status))
+	errCounter.Inc(logURL, endpoint, strconv.Itoa(status))
```

Comment:
Do you want to update the `rspsCounter` metric here?

Reply:
Updated